config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml

```diff
@@ -34,15 +34,15 @@ presets:
 
 periodics:
 - interval: 6h
-  name: ci-kubernetes-e2e-win-1-13
+  name: ci-kuberentes-e2e-aks-engine-azure-master-windows
```

Comment:
`ci-kuberentes-e2e-aks-engine-azure-master-windows` has a misspelling throughout the file